Makefile

```diff
@@ -38,6 +41,7 @@ CONDA_ACTIVATE = source $$(conda info --base)/etc/profile.d/conda.sh ; conda act
 
 ELYRA_VERSION:=$$(grep __version__ elyra/_version.py | cut -d"\"" -f2)
 TAG:=dev
+IMAGE_IS_LATEST=True
```

Comment:
I think it might be better to default this to `False` as this will prevent the accidental override of a release-relative `latest` tag with a dev-relative build using the `publish-xxx` targets directly.

Reply:
done